Python/Product/PythonTools/ptvsd/debugger.py

```diff
@@ -122,6 +97,12 @@
 # save start_new_thread so we can call it later, we'll intercept others calls to it.
 
 debugger_dll_handle = None
+
+def set_debugger_dll_handle(handle):
```

Comment:
Worth a comment that this will be called from C code, otherwise we'll regularly be trying to figure out why we have a function for this.